Geometry/CaloTopology/interface/FastTimeTopology.h

```diff
@@ -0,0 +1,121 @@
+#ifndef Geometry_CaloTopology_FastTimeTopology_h
+#define Geometry_CaloTopology_FastTimeTopology_h 1
+
+#include "DataFormats/ForwardDetId/interface/ForwardSubdetector.h"
+#include "DataFormats/ForwardDetId/interface/HGCEEDetId.h"
+#include "DataFormats/ForwardDetId/interface/HGCHEDetId.h"
+#include "DataFormats/ForwardDetId/interface/FastTimeDetId.h"
+#include "Geometry/CaloTopology/interface/CaloSubdetectorTopology.h"
+#include "Geometry/HGCalCommonData/interface/FastTimeDDDConstants.h"
+#include <vector>
+#include <iostream>
+
+class FastTimeTopology : public CaloSubdetectorTopology {
+
+public:
+  /// create a new Topology
+  FastTimeTopology(const FastTimeDDDConstants& hdcons, 
+		   ForwardSubdetector subdet, int type);
+
+  /// virtual destructor
+  virtual ~FastTimeTopology() { }  
+
+  /// move the Topology north (increment iy)  
+  virtual DetId  goNorth(const DetId& id) const {
+    return offsetBy(id,0,+1);
+  }
+  virtual std::vector<DetId> north(const DetId& id) const { 
+    DetId nextId= goNorth(id);
+    std::vector<DetId> vNeighborsDetId;
+    if (! (nextId==DetId(0)))
+      vNeighborsDetId.push_back(DetId(nextId.rawId()));
```

Comment:
so how do you propose to keep track of the issue (seems more work to remember than to fix..)